<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServicePanelMenu</class>
 <widget class="QWidget" name="ServicePanelMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0,0,0,0);
background-image:url(://ServicePanel/ServicePanel/service_panel_menu.png);
border: 0px;</string>
  </property>
  <widget class="QFrame" name="ServicePanelMenuFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="servicePanelMenuLabel">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>10</y>
      <width>566</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">SERVICE PANEL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="CalibMenuButton">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>180</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://ServicePanel/ServicePanel/calib_menu.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ToolsMenuButton">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>180</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://ServicePanel/ServicePanel/tools_menu.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PackagePanelButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/Pulsanti/Pulsanti/Clock.png);
background-image: url(:/transparent.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
